app/models/ingest/loader.rb

```diff
@@ -98,26 +98,6 @@ def load_data_from(rows, sheet_definition)
     end
     # rubocop:enable Metrics/AbcSize
 
-    def add_validation_errors!(entry, sheet_definition)
```

Comment:
Well spotted! Keeping the object method rather than a functional one will hopefully reduce the likelihood of this accidental duplication happening again :)